packages/core/src/amazonq/webview/ui/quickActions/generator.ts

```diff
@@ -25,13 +25,12 @@ export class QuickActionGenerator {
             ...(this.isFeatureDevEnabled
                 ? [
                       {
-                          groupName: 'Application Development',
                           commands: [
                               {
                                   command: '/dev',
                                   placeholder: 'Briefly describe a task or issue',
                                   description:
-                                      'Use all project files as context for code suggestions (increases latency).',
+                                      'Plan and implement new functionality across multiple files in your workspace.',
```

Comment:
We also have a [**grouping change**](https://github.com/aws/aws-toolkit-vscode/pull/4800) for the quick action commands list as requested from the UX team.

Reply:
Yes, seems that some changes overlap. I'm still waiting for a small fix for one of the texts in my PR. Can rebase mine on top of yours once you merge.